### ISSUE TITLE
Add theme support to the inspector

### ISSUE THREAD

**Issue by jacob314** [flutter/flutter-intellij#2944]
End goal is that users can understand what themes are applying to widget and can find where to go in their code to tweak the relevant themes or add new themes.

There is some existing research on analyzing what CSS styles are actually impacting how UI is rendering that could apply. The concept is to run simulations varying themes that apply to determine which themes are actually having an effect.

Currently it can take 20 minutes to track down a theme bug.  With the right tools it should be 1 minute.
@willlarche do you have any concrete examples of hand to debug theme cases? Step 1 in finding a good solution is to have some representative problem examples that we can use while developing tools and while running user studies to iterate on those tools.

**Comment by willlarche** [comment]
Yes I do. In [this episode of the Boring show](https://youtu.be/kxSVpYc2osA), we could not figure out how to change the color of text in the bottom nav and the inspector didn't give us the information we needed easily.

**Comment by willlarche** [comment]
I mean, that widget could have an easier way to set that, but it's the kind of case we care about on a big scale.

**Comment by pq** [comment]
Related to the legacy inspector; closing as stale.
